doc/source/container_builder.rst

```diff
@@ -0,0 +1,282 @@
+Building in a Self-Contained Environment
+========================================
+
+.. topic:: Abstract
+
+   Users building images with KIWI face problems if they want
+   to build as non root user or can't install KIWI on their
+   system due to inappropriate build host requirements or
+   project policies.
+
+   This document describes how to perform the build process in
+   a Docker container using the Dice containment build system
+   written for KIWI.
+
+   The changes to the machine to become a build host will
+   be reduced to the requirements of Dice and Docker.
+
+Requirements
+------------
+
+The following components needs to be installed on the build system:
+
+* Dice - a containment build system for KIWI.
+
+* Docker - a container framework based on the Linux
+  container support in the kernel.
+
+* Docker Build Container for KIWI.
```

Comment:
Isn't it a Docker image? Shouldn't it be something like:

```
* Docker Image - a Docker build container for KIWI
```

It would be consistent with the other items in the list (first the term, dash, and then the explanation).